package.json

```diff
@@ -2,7 +2,7 @@
   "name": "playcanvas",
   "displayName": "PlayCanvas",
   "description": "Official PlayCanvas extension from the team",
-  "version": "0.1.9",
+  "version": "0.1.10",
```

Comment:
We have previously published `0.1.92`. So either go `0.1.93`, or perhaps even better, `0.2.0`.